src/views/createCommunity/index.tsx

```diff
@@ -619,6 +615,14 @@ function CreateCommunityScreen() {
                 );
             }
 
+            if (coverImage !== initialData.coverImage) {
+                const apiRequestResult = await Api.upload.uploadImage(
+                    coverImage,
+                    imageTargets.COVER
+                );
+                setApiResult(apiRequestResult);
```

Comment:
Still setting the state to use below in the same function.

Reply:
Changed. Although generates more code, we're going to refactor this component anyway.